haralyzer/assets.py

```diff
@@ -286,7 +294,7 @@ def _get_asset_load(self, asset_type):
                 content_type=self.asset_types[asset_type])
 
     def filter_entries(self, request_type=None, content_type=None,
-                       status_code=None, http_version=None, regex=True):
+                       status_code=None, http_version=None, regex=True, source=None):
```

Comment:
Can we add some information in the docstring about what the `source` parameter means, and the possible options? We should probably also be checking to make sure that what they pass is either 'internal/external' and if not raise an exception.